paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -417,6 +419,17 @@ class PaymentSheet internal constructor(
          */
         val billingDetailsCollectionConfiguration: BillingDetailsCollectionConfiguration =
             BillingDetailsCollectionConfiguration(),
+
+        /**
+         * A list of preferred networks that should be used to process payments
+         * made with a co-branded card if your user hasn't selected a network
+         * themselves.
+         *
+         * The first preferred network that matches any available network will
+         * be used. If no preferred network is applicable, Stripe will select
+         * the network.
+         */
+        val preferredNetworks: List<CardBrand>? = null,
```

Comment:
To avoid exposing this while it’s still in development, we should:
1. Make this constructor internal.
2. Add a secondary constructor without `preferredNetworks` that will be the only public constructor during development.